### DISCUSSION TITLE
Using Digital Ocean Spaces for Digital Ocean Deployment

### DISCUSSION THREAD

**Discussion by daawaan4x** [Budibase/budibase#5121] [Help]
We're planning to deploy budibase on digital ocean and we'll be uploading around +3000 PDF files which are composed of scanned copies of paper documents which might take up hundreds of gigabytes. This amount of data is costly in the bundled pricing for the droplets but it is very cheap in Spaces starting at $5 only for 200GB. Since Spaces is also S3 compatible, how can we replace MinIO with Spaces instead when using the BudibaseDB?

**Comment by liongkj** [comment]
have you found a guide on this?

**Reply by daawaan4x** [reply]
Hey @liongkj, stopped using Budibase a long time ago, don't know if using other than MinIO in the self-hosted version is now allowed in newer versions of Budibase
